src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -60,6 +57,12 @@
     Text bracketCloseText;
     Text plusText;
 
+    private static final String APPEND_TO_WATCH_LIST_FORMAT = "%s;%s#%d";
+    private static final String CREATE_WATCH_LIST_FORMAT = "id:%s#%d";
```

Comment:
Eventually, we need to do more manipulations to a watch list filter. e.g. reorder, insert, delete etc.